report_qweb_signer/README.rst

```diff
@@ -0,0 +1,111 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+PDF reports signer
+==================
+
+This module extends the functionality of report addon to sign
+PDFs using a PKCS#12 certificate.
+
+
+Installation
+============
+
+To install this module, you need to install Java JDK::
+
+  apt-get install openjdk-7-jre-headless
+
+
+Configuration
+=============
+
+In order to start signing PDF documents you need to configure certificate(s)
+to use in your company.
+
+* Go to ``Settings > Companies > Companies > Your company``
```

Comment:
I don't think so. You can add a field company_dependent on the ir.actions.xml_report, and it has more sense. For example, you can want to sign an invoice, but not an invoice list you print for reporting. Both goes again same model, but different reports.